cacerts/BUILD

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Let's make this visible to just our repo for now (let's us iterate on the signature before folks depend on it).

I think this would be `//:__subpackages__`